node-electron/README.md

```diff
@@ -0,0 +1,63 @@
+# Quickstart for Electron + Agentless Rookout on AWS Lambda
+
+A sample application for using Electron + Rookout
+
+## IMPORTANT :: The supported version of electron is 1.8.7 or higher
+
+Before following this guide we recommend reading the basic [Node + Rookout] guide
+
+
+## Agent Setup
+
+
+1. Download and run the Rookout agent in a container:  
+    
+    ```bash
+    $ docker pull rookout/agent
+    $ docker run -p 7486:7486 -e "ROOKOUT_TOKEN=<Your-Token>" rookout/agent
+    ```
+
+For more information about Docker go [here](https://www.docker.com/).
+
+
+
+## Rookout Integration Explained
+
+There are 4 simple steps to integrate Rookout into your existing Node-Electron application for an [agentless] setup:
+
+1. Adding rookout to `package.json` available on [npm]:
```

Comment:
You added [npm] link but no reference to the link is in that README. Please add to the bottom the link like the other one you'll find at the bottom